### PR TITLE
Resolve MUWM-3782

### PR DESCRIPTION
**Pull request by fanglinfang**
https://jira.cac.washington.edu/browse/MUWM-3782
Built on my-test.
Testids see comment in Jira.

### CODE REVIEW

File: myuw/resources/canvas/file/api/v1/users/sis_user_id%3A12345678901234567890123456789012/enrollments_state[]_active_type[]_StudentEnrollment [hunk 1]
```diff
@@ -9,7 +9,7 @@
         "course_id": 149650,
         "sis_course_id": "2013-spring-TRAIN-100-B",
         "sis_section_id": "2013-spring-TRAIN-100-B--",
-        "html_url": "https://canvas.uw.edu/courses/149650/users/496168",
+        "html_url": "https://test.edu/courses/149650/users/496168",
```
Comment (ezturner) [inline myuw/resources/canvas/file/api/v1/users/sis_user_id%3A12345678901234567890123456789012/enrollments_state[]_active_type[]_StudentEnrollment]: should this be canvas-test.uw.edu ?

File: myuw/dao/canvas.py [hunk 2]
```diff
@@ -9,40 +9,40 @@
...
+
 def get_canvas_active_enrollments(request):
     if not hasattr(request, "canvas_act_enrollments"):
-        request.canvas_act_enrollments = _enrollments_dict_by_sws_label(
-            _get_canvas_active_enrollments_for_regid(
-                get_regid_of_current_user(request)))
+        request.canvas_act_enrollments = (
+            Enrollments().get_enrollments_for_regid(
+                get_regid_of_current_user(request),
+                {'type': ['StudentEnrollment'], 'state': ['active']}))
```
Comment (ezturner) [inline myuw/dao/canvas.py]: Is this the key change?

Comment (fanglinfang) [reply myuw/dao/canvas.py]: No change in the URL

File: myuw/resources/canvas/file/api/v1/users/sis_user_id%3A12345678901234567890123456789012/enrollments_state[]_active_type[]_StudentEnrollment (hunk 1 above)
Comment (fanglinfang) [reply myuw/resources/canvas/file/api/v1/users/sis_user_id%3A12345678901234567890123456789012/enrollments_state[]_active_type[]_StudentEnrollment]: thought we don't want to use real url

**Pull request merged**